recipe/meta.yaml

```diff
@@ -31,6 +32,7 @@ requirements:
     - tk
   run:
     - python
+    - vs2015_runtime 14.00.23026.0 0   # [win and py35]
```

Comment:
We don't want to add that dependency to our recipes, right? I believe this is something that should be fixed upstream.